charts/spiderpool/values.yaml

```diff
@@ -118,6 +118,9 @@ multus:
     ## @param multus.multusCNI.install enable install multus-CNI
     install: true
 
+    ## @param multus.multusCNI.uninstall enable remove multus-CNI configuration and binary
```

Comment:
建议 加一些 使用场景说明，是 ciilium 卸载问题， 

```
# -- Remove the CNI configuration and binary files on agent shutdown. Enable this # if you're removing Cilium from the cluster. Disable this to prevent the CNI # configuration file from being removed during agent upgrade, which can cause # nodes to go unmanageable.
```